source/content/wp-cfm.md

```diff
@@ -130,12 +143,19 @@ Deploy the `.json` file from Dev to Test.
 You can review values on the [All Settings Screen](https://codex.wordpress.org/Option_Reference#All_Settings_Screen) (`/wp-admin/options.php`).
 
 ### How can I extend WP-CFM to track more tables?
+
 If you want to track configurations in more tables, you must do so using the `wpcfm_configuration_items` hook. For details, see [WP-CFM documentation](https://forumone.github.io/wp-cfm/).
 
+### Will WP-CFM work with Multidev?
+
+Yes. For the Multidev to appear as a config option, you will need hook into the plugin's [`wpcfm_current_env`](https://github.com/forumone/wp-cfm/wiki/Filters-Reference#wpcfm_current_env) and [`wpcfm_multi_env`](https://github.com/forumone/wp-cfm/wiki/Filters-Reference#wpcfm_multi_env) fucntions, in a Must Use Plugin like the examples here setup [MU Plugin Bar](/mu-plugin/#example-code-snippets).
```

Comment:
@carl-alberto I've made some updates here, but it needs further clarification. What do you mean by "Plugin Bar"?

Would you agree with my technical edits and references to the functions?